prow/update.sh

```diff
@@ -2,22 +2,28 @@
 
 set -ex
 
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+NAMESPACE=${NAMESPACE:-default}
+WORKER_NS=${WORKER_NS:-test-pods}
+
 # safety measure: make sure we're on the right cluster
 (kubectl get nodes | grep prow-worker-01) || (echo "Wrong cluster. Exiting..."; exit 1)
 
 # make sure we use the latest configuration
 sh gen-config.sh
 
 # update config and plugins
-kubectl -n default create configmap config --from-file=config.yaml=config.gen.yaml --dry-run -o yaml | kubectl -n default replace configmap config -f -
-kubectl -n default create configmap plugins --from-file=plugins.yaml=plugins.yaml --dry-run -o yaml | kubectl -n default replace configmap plugins -f -
+kubectl -n "${NAMESPACE}" create configmap config --from-file=config.yaml=config.gen.yaml --dry-run -o yaml | kubectl -n default replace configmap config -f -
+kubectl -n "${NAMESPACE}" create configmap plugins --from-file=plugins.yaml=plugins.yaml --dry-run -o yaml | kubectl -n default replace configmap plugins -f -
 
 # update deployments etc.
-for file in cluster/*; do
-  kubectl apply -f "${file}"
+for file in "${DIR}"/cluster/*; do
+  # shellcheck disable=SC2016 ## in case of sed expression first '' is not an actual variable to be expanded
+  sed -e 's@${NAMESPACE}@'"${NAMESPACE}"'@' 's@${WORKER_NS}@'"${WORKER_NS}"'@' "$file" | kubectl apply -f -
```

Comment:
same comment about adding  the `g` to sed

Reply:
Additionally, this line was missing an extra `-e` flag - added in 9cdd333